src/lib.rs

```diff
@@ -45,8 +45,6 @@ extern crate std;
 
 #[cfg(all(feature = "alloc", not(feature = "std")))]
 use alloc::{boxed::Box, string::ToString, vec::Vec};
-#[cfg(feature = "std")]
-use std::{boxed::Box, string::ToString, vec::Vec};
```

Comment:
bcz [`std::prelude`](https://doc.rust-lang.org/std/prelude/index.html) has `Box`, `ToString`, `Vec`

Reply:
But again, in no_std, these are not included (usually) be default.

Reply:
Yes, but if built with `--all-features`, it creates warning message